index.html

```diff
@@ -267,9 +267,11 @@ <h3 class="skill-title">Skills</h3>
         </div>
     </section>
     <section class="form-section">
```

Comment:
- As per the core requirement of the task please stick to the Figma template as much as possible. Kindly left justify the content and make the font bigger and larger
- Contact form width is bigger in figma compared to your design. can you please fix that too?
- Kindly please add more padding inside the button to match the figma
- Social icons should be blue in color. Kindly fix this issue.

<img width="892" alt="Screenshot 2022-12-10 at 10 25 10 PM" src="https://user-images.githubusercontent.com/24832846/206868193-98516be1-c0bb-4bac-86d7-2ae1905cf62b.png">
<img width="1440" alt="Screenshot 2022-12-10 at 10 25 32 PM" src="https://user-images.githubusercontent.com/24832846/206868196-8d14d89d-f7b5-45dc-a276-d63bae7aff75.png">

Reply:
I have made changes to match the figma design

Reply:
> * As per the core requirement of the task please stick to the Figma template as much as possible. Kindly left justify the content and make the font bigger and larger
> * Contact form width is bigger in figma compared to your design. can you please fix that too?
> * Kindly please add more padding inside the button to match the figma
> * Social icons should be blue in color. Kindly fix this issue.
> 
> <img alt="Screenshot 2022-12-10 at 10 25 10 PM" width="892" src="https://user-images.githubusercontent.com/24832846/206868193-98516be1-c0bb-4bac-86d7-2ae1905cf62b.png">
> 
> <img alt="Screenshot 2022-12-10 at 10 25 32 PM" width="1440" src="https://user-images.githubusercontent.com/24832846/206868196-8d14d89d-f7b5-45dc-a276-d63bae7aff75.png">

- [ ] While your design looks beautiful, it would be professional to stick to the wireframe provided in the figma and left justify the content and make the font bigger and larger by giving it font-weight. 
- [ ] Take note of the contact form too, the width needs to be bigger as it is in figma. Kindly add more padding to the button to match the figma. Social icons should be blue in color. Just color: <the color code> will fix that.
### Your project
![image](https://user-images.githubusercontent.com/86200356/206875642-2053fccb-7701-4c36-a3b9-8df9c1366a86.png)

### Figma

![image](https://user-images.githubusercontent.com/86200356/206875666-106f9f94-c643-4145-8c63-2b4b151975d7.png)